File: php/oneapi/foundation/FormParameter.php
<?php

class FormParameter {
	protected $key=null;
	protected $value=null;
	
	public function getKey() { return $this->key; }
	public function getValue() { return $this->value; }
	
	public function setKey($key) { $this->key=$key; }
	public function setValue($value) { $this->value=$value; }
	
	public function __construct($key, $value) {
		$this->key=$key;
		$this->value=$value;
	}
			
}

?>

File: java/oneapi/src/org/gsm/oneapi/foundation/FormParameter.java
package org.gsm.oneapi.foundation;

/**
 *  Internally used class for storing sets of request parameters
 */
public class FormParameter {
	private String key=null;
	private String value=null;
	
	public String getKey() { return key; }
	public String getValue() { return value; }
	
	public void setKey(String key) { this.key=key; }
	public void setValue(String value) { this.value=value; }
	
	public FormParameter(String key, String value) {
		this.key=key;
		this.value=value;
	}
	
	public FormParameter() {
		
	}
	
}
